distributed/client.py

```diff
@@ -5409,8 +5409,7 @@ async def _unregister_worker_plugin(self, name, nanny=None):
 
         for response in responses.values():
             if response["status"] == "error":
-                exc = response["exception"]
-                tb = response["traceback"]
+                _, exc, tb = clean_exception(**response)
```

Comment:
Fun fact: Raising the returned exception on the client never even worked because of this.